da/avail/avail.go

```diff
@@ -398,9 +402,9 @@ func (c *DataAvailabilityLayerClient) broadcastTx(tx []byte) (uint64, error) {
 }
 
 // CheckBatchAvailability checks batch availability in DataAvailabilityLayerClient instance.
-func (c *DataAvailabilityLayerClient) CheckBatchAvailability(dataLayerHeight uint64) da.ResultCheckBatch {
+func (c *DataAvailabilityLayerClient) CheckBatchAvailability(daMetaData *da.DASubmitMetaData) da.ResultCheckBatch {
 	return da.ResultCheckBatch{
-		BaseResult: da.BaseResult{
+		BaseDACheckResult: da.BaseDACheckResult{
```

Comment:
doesn't it break all avail rollapps? they won't be able to retrieve blocks optimisticlly